src/main/java/ru/mail/dimaushenko/service/IService.java

```diff
@@ -0,0 +1,11 @@
+package ru.mail.dimaushenko.service;
+
+/**
+ *
+ * @author dmitry
+ */
+public interface IService {
```

Comment:
Better to use more readable name for an interface without I, for example HomeWorkService